__tests__/app.test.js

```diff
@@ -195,3 +195,53 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201: responds with the posted comment", () => {
+    return request(app)
+      .post(`/api/articles/:article_id/comments`)
+      .send({
+        body: "swordfish",
+        votes: 0,
+        author: "butter_bridge",
+        article_id: 1,
+        created_at: "1995-12-17T03:24:00",
+      })
+      .expect(201)
+      .then((comment) => {
+        expect(comment.text).toBe("swordfish");
+      });
+  });
```

Comment:
One test missing here: status 201 ignores unnecessary properties on the request body

Add on un needed property to the request body and check everything still works with that property ignored